<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1192</width>
    <height>725</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <weight>50</weight>
    <bold>false</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="PlotWidget" name="timeseriesgraphWidget" native="true">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>40</y>
      <width>691</width>
      <height>331</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
   </widget>
   <widget class="QPushButton" name="selectData">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>330</y>
      <width>291</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Select File</string>
    </property>
   </widget>
   <widget class="QTableView" name="tableView">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>451</width>
      <height>271</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="midLineWidth">
     <number>1</number>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAsNeeded</enum>
    </property>
    <property name="sizeAdjustPolicy">
     <enum>QAbstractScrollArea::AdjustIgnored</enum>
    </property>
   </widget>
   <widget class="QTextEdit" name="startIndex">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>350</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="endIndex">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>350</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>330</y>
      <width>161</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Start Index</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>330</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>End Index</string>
    </property>
   </widget>
   <widget class="QPushButton" name="clearAndPlotData_button">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>390</y>
      <width>291</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Reset and Plot Data</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_Update">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>390</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Update</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>320</y>
      <width>451</width>
      <height>131</height>
     </rect>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphicsView_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>460</y>
      <width>451</width>
      <height>201</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>470</y>
      <width>161</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Filters</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>500</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Savitky-Golay</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>470</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Order</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>470</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Window length</string>
    </property>
   </widget>
   <widget class="QSlider" name="savitzkyOrder">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>500</y>
      <width>121</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>11</number>
    </property>
    <property name="singleStep">
     <number>1</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="tickPosition">
     <enum>QSlider::TicksBelow</enum>
    </property>
    <property name="tickInterval">
     <number>2</number>
    </property>
   </widget>
   <widget class="QSlider" name="savitzkyLength">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>500</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>3</number>
    </property>
    <property name="maximum">
     <number>31</number>
    </property>
    <property name="singleStep">
     <number>2</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="tickPosition">
     <enum>QSlider::TicksBelow</enum>
    </property>
    <property name="tickInterval">
     <number>4</number>
    </property>
   </widget>
   <widget class="QCheckBox" name="savitzkyCheckbox">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>500</y>
      <width>21</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>540</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>High-Pass</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="highpassCheckbox">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>540</y>
      <width>21</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="labelSavitzkyOrder">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>490</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>ORDER</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelSavitzkyLength">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>490</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>LENGTH</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelHighpassCutoff">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>530</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Cutoff</string>
    </property>
   </widget>
   <widget class="QSlider" name="highpassCutoff">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>540</y>
      <width>271</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>10</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="tickPosition">
     <enum>QSlider::TicksBelow</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>580</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Detrend</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="detrendCheckbox">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>580</y>
      <width>21</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="detrendBreakpointsLineEdit">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>580</y>
      <width>251</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>580</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Breakpoints</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>600</y>
      <width>251</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>All data = 0, otherwise comma separated &quot;0,100&quot;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>620</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Asymmetric Least Squares</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="alsLineEdit">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>620</y>
      <width>251</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>1000000, 0.01</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>640</y>
      <width>251</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>10^2 ≤ λ ≤ 10^9,  0.001 ≤ p ≤ 0.1</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="alsCheckbox">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>620</y>
      <width>21</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="PlotWidget" name="loadDisplacementWidget" native="true">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>380</y>
      <width>431</width>
      <height>281</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Data loaded:</string>
    </property>
   </widget>
   <widget class="QLabel" name="currentFilePathLabel">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>10</y>
      <width>381</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="heightWidthGaplengthLineEdit">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>400</y>
      <width>251</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>1,1,1</string>
    </property>
   </widget>
   <widget class="QLabel" name="heightWidthGaplengthLabel">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>380</y>
      <width>251</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Height, Width, Gap length [mm], e.g. &quot;1,3,10&quot;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="convertToStressStrainButton">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>430</y>
      <width>121</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Stress-Strain</string>
    </property>
   </widget>
   <widget class="QPushButton" name="convertToLoadDisplacement">
    <property name="geometry">
     <rect>
      <x>1050</x>
      <y>430</y>
      <width>121</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Load-Displacement</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="youngsModulusTextEdit">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>500</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
   </widget>
   <widget class="QLabel" name="heightWidthGaplengthLabel_2">
    <property name="geometry">
     <rect>
      <x>1010</x>
      <y>500</y>
      <width>251</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Young's Modulus [N/mm²]</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="decimalSeparatorTextEdit">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>10</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;,&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>10</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Specify decimal separator:</string>
    </property>
   </widget>
   <zorder>graphicsView</zorder>
   <zorder>timeseriesgraphWidget</zorder>
   <zorder>selectData</zorder>
   <zorder>tableView</zorder>
   <zorder>startIndex</zorder>
   <zorder>endIndex</zorder>
   <zorder>label</zorder>
   <zorder>label_2</zorder>
   <zorder>clearAndPlotData_button</zorder>
   <zorder>button_Update</zorder>
   <zorder>graphicsView_2</zorder>
   <zorder>label_3</zorder>
   <zorder>label_4</zorder>
   <zorder>label_5</zorder>
   <zorder>label_6</zorder>
   <zorder>savitzkyOrder</zorder>
   <zorder>savitzkyLength</zorder>
   <zorder>savitzkyCheckbox</zorder>
   <zorder>label_7</zorder>
   <zorder>highpassCheckbox</zorder>
   <zorder>labelSavitzkyOrder</zorder>
   <zorder>labelSavitzkyLength</zorder>
   <zorder>labelHighpassCutoff</zorder>
   <zorder>highpassCutoff</zorder>
   <zorder>label_8</zorder>
   <zorder>detrendCheckbox</zorder>
   <zorder>detrendBreakpointsLineEdit</zorder>
   <zorder>label_9</zorder>
   <zorder>label_10</zorder>
   <zorder>label_11</zorder>
   <zorder>alsLineEdit</zorder>
   <zorder>label_12</zorder>
   <zorder>alsCheckbox</zorder>
   <zorder>loadDisplacementWidget</zorder>
   <zorder>label_13</zorder>
   <zorder>currentFilePathLabel</zorder>
   <zorder>heightWidthGaplengthLineEdit</zorder>
   <zorder>heightWidthGaplengthLabel</zorder>
   <zorder>convertToStressStrainButton</zorder>
   <zorder>convertToLoadDisplacement</zorder>
   <zorder>youngsModulusTextEdit</zorder>
   <zorder>heightWidthGaplengthLabel_2</zorder>
   <zorder>decimalSeparatorTextEdit</zorder>
   <zorder>label_14</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1192</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="separator"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QWidget</extends>
   <header>pyqtgraph</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
